src/tooltip/index.js

```diff
@@ -116,6 +116,20 @@ class Tooltip extends CanvasLayer {
     this.container.setTransform(x, y);
     this.container.visible = true;
   }
+  
+  getCalculatedFontSizeInPixels() {
```

Comment:
I like to practice Clean Code with function names and variables so other developers know what to expect. As the maintainer of this repo and module, feel free to change anything to be more concise and/or functional.

Reply:
I don't code that verbose normally, but that doesn't mean that it is wrong to do. It reads like old-school Java that I learned so long ago :)